sys/riscv/riscv/freebsd64_machdep.c

```diff
@@ -197,18 +197,16 @@ freebsd64_set_mcontext(struct thread *td, mcontext64_t *mcp)
 		/* XXX: Permit userland to change GPRs for sigreturn? */
 
 		/* Honor 64-bit PC. */
-		mc.mc_capregs.cp_sepcc = (uintcap_t)cheri_setoffset(
-		    (void * __capability)mc.mc_capregs.cp_sepcc,
-		    mcp->mc_gpregs.gp_sepc);
+		mc.mc_capregs.cp_sepcc = cheri_setoffset(
```

Comment:
Does it fit to re-wrap this so at least cp_sepcc is on the first line?

Reply:
The comma's in column 80, so depends who you ask.

Reply:
Hmm, is that counting the first column as 0 or 1? ;-P  If first column is 1 then I think that "fits", but it's a minor point.

Reply:
First column is 1, but I prefer to stay strictly under the 80 limit so there's space for a diff marker (though that's "free" when tabs are involved...).